src/app/components/Avatar.tsx

```diff
@@ -0,0 +1,22 @@
+import 'index.css'
+
+type AvatarProps = {
+  avatar?: string
```

Comment:
Maybe rename to `src`. I mean component is already named `Avatar`. There are no need in repetition. And also `src` can be interpreted to the src from `img` as a reference